homeassistant/components/media_player/__init__.py

```diff
@@ -811,6 +816,10 @@ def entity_picture(self):
         url = self.media_image_url
 
         if url is None:
+            if self.media_image is not None:
+                url = 'media_image'
```

Comment:
Good point. I added so that media_image also returns a hash. This way the hash only needs to be calculated once instead of with every call.